service/README.md

```diff
@@ -0,0 +1,76 @@
+# Chronon Feature Service
```

Comment:
Do you think it's worth documenting the existence of this service in the chronon website?

Reply:
Yeah def worth it. I was thinking of letting it bake / clearing out some nits etc and putting up a doc update PR in a follow up.

Reply:
Plus one to this.